scripts/build_local.sh

```diff
@@ -29,4 +29,18 @@ if [ -n "$KIND" ]; then
   kind load docker-image quay.io/operator-framework/olm:local ${KIND_FLAGS}
   kind load docker-image quay.io/operator-framework/olm-e2e:local ${KIND_FLAGS}
   kind load docker-image hang:10 ${KIND_FLAGS}
+  kind load docker-image quay.io/operatorhubio/catalog:latest ${KIND_FLAGS}
```

Comment:
This image does not exist, i.e. it was not `docker pull`d. Does this need to happen? Or does it need to be created?

Reply:
It _does_ make sense to either build these locally, or pull them down earlier to avoid quay.io timeouts, but completely fair to focus on the most pressing issues.